.github/workflows/ci.yml

```diff
@@ -660,7 +660,7 @@ jobs:
       - name: "Free space"
         run: ./scripts/ci/tools/ci_free_space_on_ci.sh
       - name: "Setup Kind Cluster ${{ env.KIND_VERSION }}"
-        uses: engineerd/setup-kind@v0.4.0
+        uses: potiuk/setup-kind@7345a7e015d9844bab5f4a8a8cfb4eb3cf7f91ef
```

Comment:
Did you pr this upstream? If so could you add a comment to the yaml saying "change back when X is merged"

Reply:
Not yet. But I plan to do it when I get it working (for now I opened the issue https://github.com/engineerd/setup-kind/issues/28). For now, it seems that the way they implemented it, we cannot even link to the commit hash for security (seems that you can only link to packaged .zip uploaded as a package which is not really what you want).
I added generated .js code back to the repo (it was .gitignored) but it's not enough - their npm run pack does not create a fully working and minified .js. I had no time to investigate but either I will make it "properly" and PR back (and they hopefully approve) or we find another action for it..

Reply:
Oh ugh :(